Dockerfile

```diff
@@ -22,6 +22,9 @@ RUN cat /etc/os-release \
         netcat \
         /tmp/rpms/openshift-clients-[0-9]*.rpm \
         /tmp/rpms/openshift-hyperkube-*.rpm \
+    && rpm-ostree override replace https://kojipkgs.fedoraproject.org//packages/kernel/6.0.18/300.fc37/x86_64/kernel-6.1.13-200.fc37.x86_64.rpm \
+                                   https://kojipkgs.fedoraproject.org//packages/kernel/6.0.18/300.fc37/x86_64/kernel-core-6.1.13-200.fc37.x86_64.rpm \
+                                   https://kojipkgs.fedoraproject.org//packages/kernel/6.0.18/300.fc37/x86_64/kernel-modules-6.1.13-200.fc37.x86_64.rpm \
```

Comment:
The URL still has old version prefix in the URL path (`6.0.18/300.fc37`).

Correct path should have this path after `kernel/`:
6.1.13/200.fc37